README.md

```diff
@@ -13,6 +13,17 @@ generates a set of random concurrent scenarios,
 examines them using either stress-testing or bounded model checking, and
 verifies that the results of each invocation satisfy the required correctness property (linearizability by default).
 
+## 📣 Package Renaming in Lincheck 2.18
```

Comment:
minor comment from outside, as next version will be:
* fully backward incompatible
* new maven artefacts
* new license

May be it's a good idea to call it lincheck 3.0? This will help users a lot during migration period :)

Reply:
That's a good point, thanks. The version "2.18" is more or less a placeholder for now, and the new release will likely be "3.0".

As we are also going to change the API dramatically, we are considering postponing this change and applying it only to the new API, keeping the current one in the "org.jetbrains.kotlinx.lincheck" package.